packages/buildcop/src/buildcop.ts

```diff
@@ -75,6 +79,139 @@ const GROUPED_MESSAGE = `Many tests failed at the same time in this package.
 
 `;
 
+interface IssuesListForRepoResponseItem {
```

Comment:
Unfortunately, @octokit/types only defines the response to this method as an array, rather than a single item. We need to define this as well for those references.

<img width="640" alt="Screen Shot 2020-10-14 at 12 29 40 AM" src="https://user-images.githubusercontent.com/55454395/95957587-bcef3600-0db4-11eb-8e1e-0a0e5e2b840d.png">